distributed/dashboard/components/scheduler.py

```diff
@@ -535,7 +535,112 @@ def update(self):
             self.fig.title.text = "Compute Time Per Task"
 
             compute_result = dict(
-                times=compute_time, color=compute_colors, names=compute_names
+                times=compute_time,
+                color=compute_colors,
+                names=compute_names,
+                formatted_time=[format_time(t) for t in compute_time],
+            )
+
+            update(self.compute_source, compute_result)
+
+
+class ComputePerKeyPie(DashboardComponent):
+    """ Pie!!!!! chart showing time spend in action by key prefix"""
```

Comment:
If we wanted to keep both I recommend that you take a look at [Bokeh tabs](http://docs.bokeh.org/en/1.0.2/docs/reference/models/widgets.panels.html) and then include two figures that are backed by the same `ColumnDataSource`